example/queue/submit_taiwania2.job

```diff
@@ -1,21 +1,21 @@
 #!/bin/bash
-#SBATCH --account=ACD108024
-#SBATCH --job-name=gamer
-#SBATCH -o log-%j
-#SBATCH --mail-type=ALL
-#SBATCH --mail-user=PUT_YOUR_EMAIL_HERE
-#SBATCH --partition=gp4d
-#SBATCH --time=2:00:00
-#SBATCH --nodes=16 --ntasks-per-node=16
-#SBATCH --gres=gpu:8 --cpus-per-task=2
-#SBATCH --ntasks-per-core=1
 
-module purge
-module load nvidia/cuda/10.1 compiler/intel/2018 openmpi3/3.1.4
+#SBATCH --job-name=YOUR_JOB_NAME          ## job name
+#SBATCH --mail-type=ALL                   ## mail notification type
+#SBATCH --mail-user=YOUR_EMAIL            ## mail address
+#SBATCH --account=YOUR_ACCOUNT            ## account (https://www.twcc.ai/user/dashboard)
+#SBATCH --partition=gp1d                  ## queue name (gtest/gp1d/gp2d/gp4d)
+#SBATCH --nodes=2                         ## number of nodes
+#SBATCH --gres=gpu:8                      ## number of GPUs per node
+#SBATCH --ntasks-per-node=8               ## number of tasks per node
+#SBATCH --cpus-per-task=4                 ## number of CPUs per task
```

Comment:
1. Does using 8 instead 4 threads per MPI process improve the performance (i.e., `--cpus-per-task=8` and `OMP_NTHREAD=8`)?
2. Does using multiple MPI processes to access the same GPU improve the performance? For example,
```
#SBATCH --gres=gpu:8                 ## number of GPUs per node
#SBATCH --ntasks-per-node=16         ## number of tasks per node
#SBATCH --cpus-per-task=2            ## number of CPUs per task
```
For the above example, one will also need to set `OMP_NTHREAD = 2` in `Input__Parameter`.

Reply:
1. There is a limit that the number of CPUs per GPU cannot be more than 4. (1 GPU : 4 CPU)
2. No, this is slightly slower.